src/ngraph_mark_for_clustering.cc

```diff
@@ -98,36 +98,47 @@ static Status TypeConstraintOk(Node* node,
 // Checks if the node meets the confirmation constraints
 static Status ConfirmationOk(
     Node* node,
-    std::map<std::string, ConfirmationFunction>& confirmation_functions,
+    std::map<std::string, ConfirmationFunction>& confirmation_function_map,
     bool& confirmation_ok) {
-  auto it = confirmation_functions.find(node->type_string());
+  auto it = confirmation_function_map.find(node->type_string());
 
-  if (it != confirmation_functions.end()) {
+  if (it != confirmation_function_map.end()) {
     TF_RETURN_IF_ERROR(it->second(node, &confirmation_ok));
   }
   return Status::OK();
 }
 
 //
-// Marks the input indices in "inputs" as static, i.e., inputs that must be
-// driven either by an _Arg or by a Const in the encapsulated graph.
+// Marks the input indices in "inputs" as static (meaning
+// that its value must be known at translation-to-nGraph time)
+//
+// Otherwise, it marks the zeroth input to the node as static (meaning
+// that its value must be known at translation-to-nGraph time, and accepts
+// placement.
 //
```

Comment:
```marks the zeroth input to the node as static ``` : Did not understand the marking of zeroth input

```accepts placement``` : but this function just sets static inputs and does nothing about accepting or rejecting nodes.

Reply:
Also the earlier comment was a defining characteristic of static input. Maybe we should retain that comment:
```
// Marks the input indices in "inputs" as static, i.e., inputs that must be
 // driven either by an _Arg or by a Const in the encapsulated graph.
```

Reply:
I did not understand the 'zeroth input' comment either. This comment was in our current version, just moved it around. I thought i had changed this comment. Forgot to save it, I guess.